data/json/mapgen/nested/cave_nested.json

```diff
@@ -0,0 +1,394 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "cave_nw_1",
+    "object": {
+      "rotation": [ 0, 0 ],
```

Comment:
Default for nested mapgen is a rotation of 0, so you shouldn't need to specify this.

Reply:
Done.